src/pypromice/postprocess/csv2bufr.py

```diff
@@ -347,31 +335,6 @@ def rolling_window(df, column, window, min_periods, decimals):
         ).median().round(decimals=decimals) # could also round to whole meters (decimals=0)
     return df
 
-def round_values(s):
```

Comment:
This resulted after a conversation with Bjarne at DMI, regarding the precision that WMO requires. Do we have some other way to ensure rounding to the specified decimal positions?

Reply:
Oh I see you moved this elsewhere in the code...